romana-install/roles/romana/kubernetes-postinstall/files/demo/demo.sh

```diff
@@ -44,10 +44,10 @@ function relative() {
     done
 }
 function get_pods() {
-    kubectl get pods -o json | jq '.items[] | { Name: .metadata.name, podIP: .status.podIP, NodeID: .spec.nodeName, Status: .status.phase }'
+    kubectl ${1:+--namespace "$1"} get pods -o json | jq '.items[] | { Name: .metadata.name, podIP: .status.podIP, NodeID: .spec.nodeName, Status: .status.phase }'
```

Comment:
Yup, except it also means if $1 wasn't provided, it's not passed as an empty value to the command, it's purely omitted.